packages/open-next/src/build/validateConfig.ts

```diff
@@ -1,26 +1,68 @@
 import {
   FunctionOptions,
+  IncludedConverter,
+  IncludedWrapper,
   OpenNextConfig,
   SplittedFunctionOptions,
 } from "types/open-next";
 
 import logger from "../logger.js";
 
+const compatibilityMatrix: Record<IncludedWrapper, IncludedConverter[]> = {
+  "aws-lambda": [
```

Comment:
Would be nice to add this matrix test in a unit test, but perhaps we can do that in the new org

Reply:
Yeah i was thinking we could do it in the new org as well.